src/wallet/rpcwallet.cpp

```diff
@@ -397,6 +398,232 @@ static void SendMoney(const CTxDestination &address, CAmount nValue, bool fSubtr
         throw JSONRPCError(RPC_WALLET_ERROR, "Error: The transaction was rejected! This might happen if some of the coins in your wallet were already spent, such as if you used a copy of wallet.dat and coins were spent in the copy but not marked as spent here.");
 }
 
+UniValue createtransaction(const UniValue& params, bool fHelp)
+{
+  if (!EnsureWalletIsAvailable(fHelp))
+      return NullUniValue;
+
+  if (fHelp || params.size() < 1 || params.size() > 2)
+      throw runtime_error(
+          "createtransaction {\"address\":amount,...} ({ \"option\": value,...})\n"
+          "\nCreate transaction. Amounts are double-precision floating point numbers."
```

Comment:
This is not correct. JSON does not have a concept of "double precision" or "floating poing", only "number". Furthermore, AmountFromValue accepts both numbers and numbers-encoded-as-strings since recently.